pydantic/fields.py

```diff
@@ -983,12 +987,14 @@ class ComputedFieldInfo:
 @typing.overload
 def computed_field(
     *,
-    return_type: Any = PydanticUndefined,
     alias: str | None = None,
     alias_priority: int | None = None,
     title: str | None = None,
     description: str | None = None,
+    examples: list[Any] | None = None,
+    json_schema_extra: JsonDict | typing.Callable[[JsonDict], None] | None = None,
     repr: bool = True,
+    return_type: Any = PydanticUndefined,
```

Comment:
I moved this so that the overload and the actual signature matched better so they were easier to compare. Then I noticed that here it's `repr: bool = True` but the actual signature has `repr: bool | None = None`, is that intentional?

Reply:
Good question.

I don't think so. I think we should use `repr: bool | None = None` here as well. As far as I understand it, the reason we allow `None` here is so that we can enforce a default of `True` if the `computed_field` is public and `False` if it's not.